docs/installation/presto-cli-installation.rst

```diff
@@ -9,18 +9,17 @@ The Presto CLI provides a terminal-based interactive shell for running queries.
 Download the `presto-cli` and copy it to the location you want to run it from. This location may be any node that has network access to the coordinator. Then rename it to presto and make it executable with chmod +x:
 ::
 
- $ mv presto-cli-0.115-executable.jar presto
+ $ mv presto-cli-*-executable.jar presto
```

Comment:
If they have multiple versions, pandemonium will ensue, especially if mv isn't aliased to mv -i. In particular, there's no telling which one will actually end up as presto, potentially leaving a version mismatch.